packages/paste-website/src/pages/tokens/design-tokens-package/index.mdx

```diff
@@ -77,17 +77,16 @@ Paste Design Tokens are supported in the following formats within this package:
 
 Paste supports multiple themes. This is done by exporting the same token formats for each theme. Each file in the root of the design tokens package is exported for each theme with the corresponding values updated for each token. This allows different parts of Twilio to have their own visual styles whilst still maintaining a consistent Twilio feel, coherent component behavior, and preventing Design System bloat. Here's an example of Themes:
 
-| Paste (default) theme            | Console theme                    | SendGrid theme                   |
-| -------------------------------- | -------------------------------- | -------------------------------- |
-| \$color-brand-highlight: #F22F46 | \$color-brand-highlight: #F22F46 | \$color-brand-highlight: #3368FA |
-| \$font-weight-semibold: 600      | \$font-weight-semibold: 500      | \$font-weight-semibold: 600      |
+| Paste (default) theme            | SendGrid theme                   |
+| -------------------------------- | -------------------------------- |
+| \$color-brand-highlight: #F22F46 | \$color-brand-highlight: #3368FA |
+| \$font-weight-semibold: 600      | \$font-weight-semibold: 600      |
```

Comment:
Lets kill this table as it's not true anymore. Plus the `Here's an example of Themes:` text above it.